vue/components/ui/molecules/filter/filter.vue

```diff
@@ -135,6 +139,9 @@ export const Filter = {
                 start: this.start,
                 limit: this.limit
             };
+        },
+        scopedSlots() {

```

Comment:
It says `scopedSlots` but in fact it uses `$slots`. I'm confused :p

Reply:
Bad naming on my part, sorry